### PR TITLE
feat(cli): add workspaces and contributors to the sort order

### PR DESCRIPTION
**Pull request by camacho**
This PR:
- adds "workspaces" and "contributors" to the order list
- adds `CHANGELOG`
- updates dependencies
- adds back in previously excluded `__tests__` directory
- updates tests to use `toMatchObject` instead of checking each property individually
- use `toMatchInlineSnapshot` in favor of `toMatchSnapshot` for better readability and debugging when testing
- uses `resolves`/`rejects` where possible in tests instead of `await`ing
- updates error messages to end with periods

### CODE REVIEW

File: lib/cli/config.js [hunk 1]
```diff
@@ -18,12 +18,15 @@ const configDefault = {
 
 const configValidate = config => {
   const result = Joi.validate(config, JoiConfigSchema);
+
   if (result.error) return result;
+
   if (!Array.isArray(config.order)) {
     return {
-      error: new ValidationError('empty order property'),
+      error: new ValidationError('Empty order property.'),
```
Comment (peterkc) [inline lib/cli/config.js]: Not sure what the common pattern is for error messages, in regards to ending with a `.` or not.

File: lib/cli/error.spec.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ describe('error', () => {
       value: jest.fn(),
     });
 
-    error = new Error('Standard message');
+    error = new Error('Standard message.');
```
Comment (peterkc) [inline lib/cli/error.spec.js]: Same as previous.

File: scripts/markdown-transformers.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ function REGION(content, options, config) {
     code = code.match(pattern);
     code = code[0].replace(pattern, '$2');
   } else {
-    throw new Error(`Region not found: ${region}`);
+    throw new Error(`Region not found: ${region}.`);
```
Comment (peterkc) [inline scripts/markdown-transformers.js]: Same as previous.

**Comment by camacho** [comment]
Thanks for the review! For the error messages, I went with sentence format because the message gets printed.

**Pull request merged**